eng/service.proj

```diff
@@ -47,7 +47,7 @@
   <Target Name="GenerateCode">
     <MSBuild Projects="@(ProjectReference)"
              Targets="GenerateCode"
-             BuildInParallel="$(BuildInParallel)"
+             BuildInParallel="false"
```

Comment:
Not sure this was an intentional inclusion, as it potentially impacts all builds for all packages.   

If this was intentional, please help us understand the rationale behind the change, the expected impact, and how we should test the change.

Reply:
Per offline conversation, this is the best workaround available as this impacts multiple libraries and will only non-parallelize the code generation step.